flake8_print.py

```diff
@@ -1,54 +1,40 @@
-import ast
-import tokenize
+"""Extension for flake8 that finds usage of print."""
+import re
 
-from sys import stdin
-
-__version__ = '1.6.1'
+__version__ = '2.0.0'
 
 PRINT_ERROR_CODE = 'T001'
 PRINT_ERROR_MESSAGE = 'print statement found.'
 
 
-class PrintStatementChecker(object):
-    name = 'flake8-print'
-    version = __version__
-
-    def __init__(self, tree, filename='(none)', builtins=None):
-        self.tree = tree
-        self.filename = (filename == 'stdin' and stdin) or filename
-
-    def run(self):
-        if self.filename == stdin:
-            noqa = get_noqa_lines(self.filename)
-        else:
-            with open(self.filename, 'r') as file_to_check:
-                noqa = get_noqa_lines(file_to_check.readlines())
-
-        errors = check_tree_for_print_statements(self.tree, noqa)
-
-        for error in errors:
-            yield (error.get("line"), error.get("col"), error.get("message"), type(self))
+def flake8ext(f):
```

Comment:
this isnt necessary, and could be removed..? (I copied the idea from the `hacking` package)
and it should probably be a private function..?